app/database/connection.py

```diff
@@ -0,0 +1,13 @@
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+
+
+class Connection():
+    _Session = None
+
+    @classmethod
+    def get(cls, url):
+        if cls._Session is None:
+            engine = create_engine(url, echo=False)
+            cls._Session = sessionmaker(bind=engine)
+        return cls._Session()
```

Comment:
I'm not sure what you meant by this comment. When were we talking about disallowing the ability to create new sessions?